addon/components/date-range-picker.js

```diff
@@ -48,6 +48,8 @@ export default Ember.Component.extend({
   cancelLabel: 'Cancel',
   applyAction: null,
   cancelAction: null,
+  autoUpdateInput: false,
+  autoApply: false,
```

Comment:
Apologies -- `autoUpdateInput` now has the correct default value. `autoApply` was already correct.